### DISCUSSION TITLE
Losing sensors and speed control

### DISCUSSION THREAD

**Discussion by TatsuS12** [EvanMulawski/FanControl.CorsairLink#48] [General]
As soon as I open SignalRGB v2.2.31.0 @ f85f9017 FanControl loses all input and output from my Corsair Commander Pro and it sounds like speeds default to 50%. Was hoping this would be a better and more configurable solution to needing to restart icue everytime i woke my pc up, but having to choose between rgb control or fan control makes icue look like the much better option still

**Comment by EvanMulawski** [comment]
Hi @TatsuS12, can you post the `log.txt` file located in the Fan Control root directory?

SignalRGB 2.2.29+ is compatible with this plugin and others are using this combination successfully.

**Comment by TatsuS12** [comment]
The log only shows that the core and core xt couldn't initialise, since I don't have either of these devices but have the commander pro. I have taken a couple of short clips showing what is happening both without signalrgb open and it being open

[log.txt](https://github.com/EvanMulawski/FanControl.CorsairLink/files/11581241/log.txt)

**Reply by EvanMulawski** [reply]
I believe you are using a different plugin - the developer deleted the repository and the plugin is not stable.

You can use the CorsairLink plugin by following the instructions in the [Installation](https://github.com/EvanMulawski/FanControl.CorsairLink#installation) guide for this repository. (And make sure to remove the DLL for the other plugin.)

**Reply by TatsuS12** [reply]
Ahh, i did download via a youtube vid before I went hunting better and found this, will double check I have the right 1 installed cause yes very possible I used the wrong plug in. I just know this is the same behaviour that icue shows if any other monitoring software is running

**Reply by TatsuS12** [reply]
That was indeed 1 problem and has been fixed, but now, instead of fan control loosing control of the fans and sensors, the sensor and fan reading bounce all over the place and sometimes completely loosing the sensor signal, rather than remaining stable. 

I experienced this exact behaviour using icue and any other rgb/hardware monitoring software at the same time and was hopeing this would avoid that, as when the fan curve looses the water temp reading it ramps up and down rapidly

**Reply by EvanMulawski** [reply]
That definitely sounds like an incompatible program is communicating with the device. Please post your log.txt file again, make sure the Corsair Service is not running, and stop any other software that might be communicating with the device. Only programs that implement a well-known lock to synchronize device communication, such as HWiNFO and SignalRGB, are supported. 

Note: I have a Commander Pro and use SignalRGB and HWiNFO without issue.

**Comment by TatsuS12** [comment]
Its a fresh windows11 install, icue hasnt been installed, no other monitoring software is running, nor any other rgb control software currently. I thought maybe logitech's Ghub, but stopped it and restarted both fan control and signalrgb with no change in behaviour. the only other piece of corsair hardware I have connected is thier k95 keyboard

[log.txt](https://github.com/EvanMulawski/FanControl.CorsairLink/files/11588797/log.txt)
